projects/angular-tags-input/src/lib/dropdown/dropdown.component.ts

```diff
@@ -106,9 +107,14 @@ export class DropdownComponent
    */
 
   filterItems(searchTerm = this.inputVal, items = this.listItems) {
-    this.context.items = [
-      ...this.dropdownItemsFilter.transform(items, this.config, searchTerm)
-    ];
+    const filteredItems = this.dropdownItemsFilter.transform(items, this.config, searchTerm);
+    if (filteredItems.length === 0) {
+      this.context.items = [];
+      this.noRecordsFound = true;
+    } else {
+      this.context.items = [...filteredItems];
+      this.noRecordsFound = false;
+    }
```

Comment:
**issue:** I think this is not necessary, please see my comment on the HTML template